manifests/leader-worker-set.clusterserviceversion.yaml

```diff
@@ -0,0 +1,456 @@
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - leaderworkerset.x-k8s.io
+          resources:
+          - leaderworkersets/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - leaderworkerset.x-k8s.io
+          resources:
+          - leaderworkersets/status
+          verbs:
+          - get
+          - patch
+          - update
+        - nonResourceURLs:
+          - /metrics
+          verbs:
+          - get
+        - apiGroups:
+          - authentication.k8s.io
+          resources:
+          - tokenreviews
+          verbs:
+          - create
+        - apiGroups:
+          - authorization.k8s.io
+          resources:
+          - subjectaccessreviews
+          verbs:
+          - create
+        - apiGroups:
+          - operator.openshift.io
+          resources:
+          - leaderworkersetoperators
+          - leaderworkersetoperators/status
+          verbs:
+          - create
+          - delete
+          - deletecollection
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - clusterroles
+          - clusterrolebindings
+          - roles
+          - rolebindings
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - admissionregistration.k8s.io
+          resources:
+          - validatingwebhookconfigurations
+          - mutatingwebhookconfigurations
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - apiextensions.k8s.io
+          resources:
+          - customresourcedefinitions
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        serviceAccountName: openshift-lws-operator
+      deployments:
+      - name: openshift-lws-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: openshift-lws-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: openshift-lws-operator
+            spec:
+              containers:
+              - args:
+                - operator
+                command:
+                - lws-operator
+                env:
+                - name: OPERATOR_NAME
+                  value: openshift-lws-operator
+                - name: RELATED_IMAGE_OPERAND_IMAGE
+                  value: ${OPERAND_IMAGE}
+                image: ${OPERATOR_IMAGE}
+                imagePullPolicy: Always
+                name: openshift-lws-operator
+                ports:
+                - containerPort: 60000
```

Comment:
Do we use this port for metrics? And any specific reason why is it this number? Could be useful to have 80 or 443 suffix.

Reply:
It is auto-generated by `operator-sdk generate bundle` command. But good point, I'll check what it is.

Reply:
That is a nice catch. Library-go https://github.com/openshift/library-go/blob/ac3ba9eb16a23890629f719f3fe11428ad395796/pkg/config/configdefaults/config_default.go#L46 defaults to 8443 as metrics server port. I've updated deployment.yaml and CSV file with `8443`.

Reply:
Cool, thx for the update!